src/Mvc/Mvc.Analyzers/src/Microsoft.AspNetCore.Mvc.Analyzers.csproj

```diff
@@ -9,6 +9,7 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
     <NuspecFile>$(MSBuildProjectName).nuspec</NuspecFile>
+    <NullableContextOptions>enable</NullableContextOptions>
```

Comment:
Also set `<Nullable>enable</Nullable>` the compiler has already renamed this property, and it will just start failing the build due to not recognizing `IPropertySymbol?` when it hits us.